app/models/account.rb

```diff
@@ -130,6 +130,10 @@ def mute!(other_account)
     mute_relationships.find_or_create_by!(target_account: other_account)
   end
 
+  def mute_conversation!(conversation)
+    ConversationMute.find_or_create_by!(account: self, conversation: conversation)
```

Comment:
Since this is inside `Account`, could this just be `conversation_mutes.where(conversation: conversation).first_or_create!` (or similar) to avoid the use of `self` here?